actioncable/lib/action_cable/server/broadcasting.rb

```diff
@@ -46,7 +39,7 @@ def initialize(server, broadcasting)
 
           def broadcast(message)
             server.logger.info "[ActionCable] Broadcasting to #{broadcasting}: #{message}"
-            server.broadcasting_redis.publish broadcasting, ActiveSupport::JSON.encode(message)
+            server.pubsub.broadcast broadcasting, ActiveSupport::JSON.encode(message)
```

Comment:
Actually, this is the only place that mistake is made. We need to use the broadcasting redis here. Not the evented pubsub redis.